packages/eslint-config/core.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:prettier/recommended"],
+    parser: "@typescript-eslint/parser",
+    plugins: ["@typescript-eslint", "simple-import-sort", "unused-imports", "import"],
+    rules: {
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": ["error", { ignoreRestSiblings: true }],
+        "prefer-template": "error",
+        "simple-import-sort/exports": "error",
+        "simple-import-sort/imports": "error",
+        "unused-imports/no-unused-imports": "error",
+        "import/no-extraneous-dependencies": "error",
+        "no-duplicate-imports": "error",
+        "no-console": "error",
+    },
+    overrides: [
+        {
+            files: ["*.ts", "*.tsx"],
+            rules: {
+                "@typescript-eslint/no-unused-vars": ["error", { args: "none", ignoreRestSiblings: true }],
+            },
+        },
+    ],
+};
```

Comment:
there are slight differences with https://gitlab.vivid-planet.com/vivid/comet/-/blob/main/eslint-config/core.js, why that?

Reply:
only one change and thats because of these codes we have in multiple files:
https://github.com/vivid-planet/comet-admin/blob/setup-eslint-package/packages/admin/admin/src/table/TableFilterFinalForm.tsx#L20